addons/event/models/event_ticket.py

```diff
@@ -47,6 +49,7 @@ class EventTicket(models.Model):
     _name = 'event.event.ticket'
     _inherit = 'event.type.ticket'
     _description = 'Event Ticket'
+    _order = 'sequence, id'
```

Comment:
I think using the ticket's name would also make sense.
```suggestion
    _order = 'sequence, name, id'
```

Reply:
Good point. Just a thought here. As the name, sequence ... are copied on the event tickets when using an event template (=> Event Template Tickets), I've also added the name in the order on event.type.ticket to have the same behavior.

Therefore, as event.event.ticket inherits from event.type.ticket, it is not strictly necessary to have a line to define _order= since the ordering is the same.

However, I've still kept it, to have an explicit declaration, since there is further inheritance going on. Indeed, we may have a difference at some point in the ordering in submodules. What do you think?

Cheers!

Reply:
I'm wondering now if the event name/id/something shouldn't be part of ordering for the ticket as well, as you may have several `event.ticket`s for different `event`s coming from the same `event.ticket.type`. In a list of tickets, I suppose it would make sense to have
```txt
Event 1 | Ticket 1
Event 1 | Ticket 2
Event 2 | Ticket 1
Event 2 | Ticket 2
```
Rather than
```txt
Event 1 | Ticket 1
Event 2 | Ticket 1
Event 1 | Ticket 2
Event 2 | Ticket 2
``` 
In which case we already have a difference between `event.ticket.type` and `event.ticket` `_order`s (and the latter could use a `event.type` field to differentiate similar `event.tickets.types`?). What do you think?

Reply:
Interesting. Just to be sure, is there any situation where this could happen? An event can only have one Template, erasing the current list of tickets, (and copying sequence) hence preserving the order. When configuring products, the tickets are filtered to only be the ones linked to the selected event.

Is there somewhere I missed where a single list could incorporate tickets from several events? maybe in the front end? Cheers

Reply:
I haven't found it indeed, I had the `survey.question` case in mind which does (and is somewhat unordered).
The event case isn't visible even on the implemented reporting, so we can skip this for now.

I would prefer not to repeat the `_order` if it's the same as the parent. Otherwise, we'll have two lines to maintain to do one change (in addition to the extensions if they do override it) or unexpected behaviors/errors if we forget to do it.